src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1026,9 +1248,64 @@ private static GencodeFuncotation createIntronFuncotation(final VariantContext v
             );
         }
 
+        // Set our version:
+        gencodeFuncotationBuilder.setVersion(version);
+
         return gencodeFuncotationBuilder.build();
     }
 
+    /**
+     * Get the bases around the given variant (as specified by {@code refAllele} and {@code altAllele}).
+     * The number of bases before and after the variant is specified by {@link #referenceWindow}.
+     * @param refAllele The reference {@link Allele} for the variant.
+     * @param altAllele The alternate {@link Allele} for the variant.
+     * @param reference The {@link ReferenceContext} for the variant, with the current window around the variant.
+     * @param strand The {@link Strand} on which the variant occurs.
+     * @return A {@link String} of bases of length {@link #referenceWindow} * 2 + |variant|
+     */
+    private static String getReferenceBases(final Allele refAllele, final Allele altAllele, final ReferenceContext reference, final Strand strand ) {
+
+        // Adjustment for the start of the variant based on whether it's a deletion or not.
+        // NOTE: This is an artifact of the GATK representation of variants (that they must be at least 1 base)
+        final int deletionStartAdjustment =  GATKVariantContextUtils.isDeletion(refAllele, altAllele) ? 1 : 0;
```

Comment:
this is unused, I assume it's supposed to be used in here somewhere

Reply:
No... this is leftover from a previous fix for the reference context field.  I'll remove it.